content/SCALE/GettingStarted/Migrate/MigratePrep.md

```diff
@@ -8,76 +8,115 @@ tags:
+   <input type="checkbox"> Data protection tasks - Record or screenshot replication, periodic snapshot, cloud sync, or other tasks settings to reconfigure these in SCALE if you want to duplicate these tasks. 
 
-   * CORE init/shutdown scripts to add to SCALE.
+4. Write down your network configuration information or take screenshots of the global network settings, interfaces, static IP addresses, and aliases configured.
+   
+   FreeBSD and Linux use different nomenclature for network interfaces, bridges, LAGGs, and VLANs. 
+   Because of the difference, network settings can either get lost or not transfer which means you have no network connectivity.
+   You can find interface names in the CORE UI on the **Network > Interfaces** screen.
 
-   * CORE cron jobs configured if you want to set the same jobs up in SCALE.
+   If using TrueNAS System servers, to find the Linux interface ID equivalent, refer to the network port ID manuals for your model found in the [Solutions > TrueNAS Systems](https://www.truenas.com/docs/hardware/) area of the Documentation Hub for network port assignments.   
+   If not using TrueNAS system hardware, refer to the manual or documentation provided with your system or locate this information on your server hardware and take note of it.
 
-   * The global self-encrypting drive (SED) password to configure in SCALE, or unlock these drives in CORE before you clean install SCALE.
+   If there are issues after a clean install of SCALE from an <file>iso</file> file or you are not using DHCP for network and interface configuration, use the information from your CORE settings to configure your SCALE network settings and to reconfigure your static IPs or aliases.
+      {{< include file="/_includes/NetworkInstallRequirementsSCALE.md" >}}
 
-   * Cloud storage backup provider credentials configured in CORE if you do not have these recorded in other files kept secured outside of CORE.
+5. Migrate the deprecated S3 MinIO service (if in use). See [services deprecated in SCALE](#migrating-from-deprecated-services).
+   Read and follow instructions in [Migrating from MinIO S3](https://www.truenas.com/docs/scale/22.12/scaletutorials/apps/communityapps/minioclustersscale/migratingfroms3service/)!
+   Make sure S3 MinIO data is backed up as a precaution. The S3 service requires installing SCALE 22.12.3 (Bluefin).
+   This version of SCALE provides access to both the S3 service and the MinIO app you migrate to. 
 
-   * Replication, periodic snapshot, cloud sync, or other tasks settings to reconfigure in SCALE if you want to duplicate these tasks.
+6. Back up your stored data files.
+   If you need to do a clean install with the SCALE <file>iso</file> file, you can import your data pools into SCALE.
+   Verify you can access your backed up filed before you upgrade/migrate to SCALE.
 
-   * Make sure you have backed-up copies of certificates used in CORE to import or configure in SCALE.
+7. Back up any critical data. 
```

Comment:
6. and 7. are essentially the same thing, go ahead and combine.

Reply:
Combined in the next commit.